front/app/api/background_jobs/useTrackBackgroundJobs.ts

```diff
@@ -7,7 +7,7 @@ import useBackgroundJobs from './useBackgroundJobs';
 
 const useTrackBackgroundJobs = ({
   jobs,
-  onChange, // Make sure to use `useCallback` for `onChange`
+  onChange, // Make sure onChange is not changed on every render (e.g., use `useCallback` around it where you call useTrackBackgroundJobs)
```

Comment:
It's generally better to come up with a different mechanism for this that doesn't require `useCallback` every time this hook is called but since this was already the case, making the comment clearer is a good step. 👍